persistence/src/main/scala/hmda/persistence/processing/HmdaRawFile.scala

```diff
@@ -31,6 +34,8 @@ object HmdaRawFile {
 
 class HmdaRawFile(submissionId: SubmissionId) extends HmdaPersistentActor {
 
+  var fileName: String = ""
```

Comment:
I'm curious about the strategy here. What are the pros and cons of using a `var` to store the `fileName` rather than storing it in the `HmdaRawFileState` object?